common/dna_data/wc_orc_dna.txt

```diff
@@ -1893,3 +1893,123 @@ griselda_dna = { # by icelille999
 	}
 	enabled=yes
 }
+grommash_dna = { # by Herbe
```

Comment:
You forgot to assign beards and hairs. From the 11th step.
<https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/How-To-Assign-DNAs-(Portraits)-To-Characters>
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/123955487-b7756280-d9ba-11eb-8d5a-eb4ed1c7f98a.png)

</details>